server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/DatafileManager.java

```diff
@@ -500,8 +519,8 @@ public int getNumFiles() {
     return datafileSizes.size();
   }
 
-  public long getUpdateCount() {
-    return updateCount;
+  public MetadataUpdateCount getUpdateCount() {
+    return new MetadataUpdateCount(metadataUpdatesStarted.get(), metadataUpdatesCompleted.get());
```

Comment:
I have a slight concern about this code.  It may matter that the start count is read before the completed count, not sure still thinking out it.  Thinking of replacing the two separate atomic longs to a single atomic reference to a counter object with two longs.  Then I don't have to worry about this issue.

Reply:
changed this in fd31a15